packages/nextra-theme-docs/css/styles.css

```diff
@@ -246,7 +255,7 @@ input[type='search']::-webkit-search-results-decoration {
   display: none;
 }
 .locale-switch[open] summary:before {
-  content: ' ';
+  content: '';
```

Comment:
this class seems not used anywhere, not?

Reply:
Oh right, that seems to be legacy